src/models/patreonId.js

```diff
@@ -2,13 +2,14 @@ import mongoose from 'mongoose';
 
 // SCHEMA SETUP
 const patreonIdSchema = new mongoose.Schema({
```

Comment:
```suggestion
const patreonRecordSchema = new mongoose.Schema({
```

Let's rename this collection.